cmd/pbapi/main.go

```diff
@@ -61,14 +61,14 @@ func main() {
 	mr.Get("/", statusOk)
 	mr.Handle("/metrics", promhttp.Handler())
 
-	log.Info().Msg("Starting new instance on 8000/8080")
+	log.Info().Msg("Starting new instance on 8000/9000")
 	srv := http.Server{
 		Addr:    fmt.Sprintf(":%d", 8000),
 		Handler: r,
 	}
 
 	msrv := http.Server{
-		Addr:    fmt.Sprintf(":%d", 8080),
+		Addr:    fmt.Sprintf(":%d", 9000),
```

Comment:
Can you extract these into viper variables? When running without containers ports can be occupied by other processes.